### ISSUE TITLE
[BUG] Failed to enable vm-importer, pcidevices and harvester-seeder controller addons, keep stuck in "Enabling" state

### ISSUE THREAD

**Issue by TachunLin** [harvester/harvester#5337]
**Describe the bug**

On a clean installed single or four nodes v1.3.0-rc4 Harvester cluster.  
When we try to enable the `vm-import-controller`, `pcidevices-controller` and `harvester-seeder` on the Harvester `Addons` page.

The enabling process would keep in the `Enabling` state with failure message
```
addon deployment job helm-install-vm-import-controller failed
addon deployment job helm-install-pcidevices-controller failed
addon deployment job helm-install-harvester-seeder failed
```

![image](https://github.com/harvester/harvester/assets/29251855/674e3d0c-b13a-4642-b53f-353f88b50a33)

* Check the related job log of `helm-install-pcidevices-controller` , we could see the following message
```
 + helm_v3 install --version 0.3.2 pcidevices-controller pcidevices-controller/harvester-pcidevices-controller --values /config/values-01_HelmChart.yaml    │
│ Error: INSTALLATION FAILED: chart "harvester-pcidevices-controller" matching 0.3.2 not found in pcidevices-controller index. (try 'helm repo update'): no  │
│ chart name found
```

* The related jobs could not be completed
![image](https://github.com/harvester/harvester/assets/29251855/cae1cad7-079f-4c9a-812d-d786ea4ebde2)

**To Reproduce**
1. Open Addons page
2. Enable the vm-import-controller addons, 
3. Check until the state changes
4. Access Harvester node 
5. Use K9s or kubectl to check the `helm-install-vm-import-controller` job state and logs
6. Enable the pcidevices-controller and harvester-seeder addons
7. Check until the state changes
8. Use K9s or kubectl to check the `helm-install-pcidevices-controller` and `helm-install-harvester-seeder` job state and logs

**Expected behavior**

Should be able to enable all addons on the Harvester Addons page in the running state without failure

**Support bundle**

* Part1: (4 nodes cluster)
[supportbundle_d9badc09-64f3-492c-bc4d-d8163adb763f_2024-03-11T07-41-00Z.zip](https://github.com/harvester/harvester/files/14555082/supportbundle_d9badc09-64f3-492c-bc4d-d8163adb763f_2024-03-11T07-41-00Z.zip)

* Part2: (single node cluster)
[hp-164supportbundle_d9badc09-64f3-492c-bc4d-d8163adb763f_2024-03-11T03-33-06Z.zip](https://github.com/harvester/harvester/files/14555045/hp-164supportbundle_d9badc09-64f3-492c-bc4d-d8163adb763f_2024-03-11T03-33-06Z.zip)

**Environment**
 - Harvester ISO version: v1.3.0-rc4
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): Single and Four nodes baremetal machines

**Additional context**
The `rancher-logging`, `rancher-monitoring` and `nvidia-driver-toolkit` addons are **not** affected

**Comment by w13915984028** [comment]
Looks like the shell script has a fault to skip arm; I will fix this.

../scripts/build-bundle#L107C1-L124C3
```
# skip addons for now for arm builds
if [ "{ARCH}" == "amd64" ]  /////////// here
then
# Prepare vm-import-controller-chart
helm pull https://github.com/harvester/charts/releases/download/harvester-vm-import-controller-${VM_IMPORT_CONTROLLER_CHART_VERSION}/harvester-vm-import-controller-${VM_IMPORT_CONTROLLER_CHART_VERSION}.tgz -d ${CHARTS_DIR}
# make chart sanity check
tar zxvf ${CHARTS_DIR}/harvester-vm-import-controller-${VM_IMPORT_CONTROLLER_CHART_VERSION}.tgz >/dev/null --warning=no-timestamp

...
fi
```

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.3-f8469881-head` (24/03/12). Close this issue.

### Result
<details><summary>$\color{green}{\textsf{PASS}}$ Single node kvm enable virt-import controller $~~$</summary>

* Can enable the `virt-import controller` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/6cfbdafa-4aa0-4873-ad88-9fa3b29ffeba)

![image](https://github.com/harvester/harvester/assets/29251855/9f833c9b-730a-4e88-93b7-3890fc08e30f)


![image](https://github.com/harvester/harvester/assets/29251855/49894557-36a9-41d3-9b88-3f24a89f4eef)

 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Single node kvm enable pcidevices-controller $~~$</summary>

* Can enable the `pcidevices-controller` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/4dd4dff6-aa79-49a5-be41-cee42e48b36d)

![image](https://github.com/harvester/harvester/assets/29251855/8e796493-13d4-41b8-9482-03413fca8aa3)


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Single node kvm enable harvester-seeder $~~$</summary>

* Can enable the `harvester-seeder` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/b13e8362-306f-495a-a22f-1e6bbd87cb5a)

![image](https://github.com/harvester/harvester/assets/29251855/62e26c3d-0c79-4501-956e-e7207668a3a0)


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Single node kvm enable nvidia-driver-toolkit, rancher-logging and rancher-monitoring $~~$</summary>

* Can enable the `nvidia-driver-toolkit`, `rancher-logging`, `rancher-monitoring ` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/939966c9-b258-4974-8c85-4cc24b119730)

![image](https://github.com/harvester/harvester/assets/29251855/54b25be9-9651-4a63-95d8-bec1ad3a440a)

![image](https://github.com/harvester/harvester/assets/29251855/4d2e03aa-1240-4892-b88d-2a36a92bd689)

![image](https://github.com/harvester/harvester/assets/29251855/f576f3d5-1c0f-44ba-995b-0cfe00e3e0e6)


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Three nodes baremetal machines enable virt-import controller $~~$</summary>

* Can enable the `pcidevices-controlle` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/6a65d81f-20fa-4cc4-818a-0ccc9c06f8a5)

![image](https://github.com/harvester/harvester/assets/29251855/b95097c7-3e15-454f-8182-34e80fcdc709)


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Three nodes baremetal machines enable pcidevices-controller $~~$</summary>

* Can enable the `pcidevices-controller` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/f7b3f440-8a0a-4551-a2ca-6d126efd0468)

![image](https://github.com/harvester/harvester/assets/29251855/908fcfbe-bc90-427d-9cc3-293c54fd4939)


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Three nodes baremetal machines enable harvester-seeder $~~$</summary>

* Can enable the `harvester-seeder` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/5e2faba2-b609-4553-939e-c62b7c8a23d3)

![image](https://github.com/harvester/harvester/assets/29251855/899a48a3-9f5e-4b58-8bcc-467d9b80660d)

 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Three nodes baremetal machines enable nvidia-driver-toolkit, rancher-logging and rancher-monitoring  $~~$</summary>

* Can enable the `nvidia-driver-toolkit`, `rancher-logging`, `rancher-monitoring ` and helm install chart on jobs

![image](https://github.com/harvester/harvester/assets/29251855/fd54e860-aa18-4004-a9de-c034855a0b35)

![image](https://github.com/harvester/harvester/assets/29251855/2e4052aa-8f63-4d3c-bc98-2cc967613f85)

![image](https://github.com/harvester/harvester/assets/29251855/85c8e8d0-1dd9-46e1-ab5d-7e51dd4afcd1)

![image](https://github.com/harvester/harvester/assets/29251855/ca88c1d6-3cbd-40fe-9015-9527526ac2d4)


 </details>

### Test Information
* Test Environment: Single nodes on local kvm, three nodes baremetal machines
* Harvester version: `v1.3-f8469881-head` (24/03/12)

### Verify Steps
<details><summary>Enable addons on single and three nodes cluster </summary>

  1. Prepare a single and three nodes Harvester cluster
  2. Open Addons page
  3. Access to harvester node machine 
  4. Open k9s
  5. Enable the `vm-importer`, `pci-devices` and `harvester-seeder` addons
  6. Check the corresponding jobs and logs
  7. Enable rest of the addons `nvidia-driver-toolkit`, `rancher-monitoring` and `rancher-logging`

</details>
